hack/rpm-deps.sh

```diff
@@ -30,6 +30,7 @@ fedora_extra="
   libcurl-minimal
 "
 centos_base="
+  acl
```

Comment:
You're adding the package to $centos_base instead of $handler_base, which means it will show up in basically all containers. I'm not sure whether that's what you wanted to achieve; if so, you need to update the commit message accordingly.

Reply:
Clarified the commit message. The issue is that without this package inside the containers I can't build any of these. This was already the case when we were still on fedora but it was in every container and therefore we never saw an issue. I think I will investigate this further but it works for now.

Reply:
The commit's subject still says "Add acl package to virt-handler".

Everything else looks great :)

Reply:
Now it should really be fine.